x/staging/posts/client/cli/cli_test.go

```diff
@@ -838,20 +838,37 @@ func (s *IntegrationTestSuite) TestCmdReportPost() {
 		{
 			name:   "invalid post id",
 			expErr: true,
-			args:   []string{"1", "scam", "message"},
+			args:   []string{"1", "message"},
 		},
 		{
 			name:   "invalid report type",
 			expErr: true,
-			args:   []string{"a56145270ce6b3bebd1dd012b73948677dd618d496488bc608a3cb43ce3547dd", "", "message"},
+			args: []string{
+				"a56145270ce6b3bebd1dd012b73948677dd618d496488bc608a3cb43ce3547dd",
+				"message",
+				fmt.Sprintf("--%s=%s", cli.FlagReason, ""),
+			},
 		},
 		{
 			name:   "valid report",
 			expErr: false,
 			args: []string{
 				"a56145270ce6b3bebd1dd012b73948677dd618d496488bc608a3cb43ce3547dd",
-				"scam",
 				"message",
+				fmt.Sprintf("--%s=%s", cli.FlagReason, "scam"),
+				fmt.Sprintf("--%s=%s", flags.FlagFrom, val.Address.String()),
+				fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
+				fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+				fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
+			},
+			respType: &sdk.TxResponse{},
+		},
+		{
```

Comment:
Can we add a test to make sure that using multiple times the `--reason` flag works properly?